addons/l10n_tw_edi_ecpay_pos/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    "name": "Taiwan - E-invoicing Pos",
+    "countries": ["tw"],
```

Comment:
I believe the countries key should only be set on base modules, these bridge modules don't need it